css/style.css

```diff
@@ -0,0 +1,1027 @@
+  color: black;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding-top: 18px;
+  padding-bottom: 0;
+  padding-right: 10px;
+  padding-left: 10px;
+}
+
+header > div.headersecond-row > nav,
+header > div,
+div.headersecond-row nav ul > li {
+  display: inline-block;
+}
+
+div.headersecond-row {
+  display: block;
+  height: 3em;
+}
+
+div.headersecond-row > div {
+  float: right;
+  display: flex;
+  margin-right: 1em;
+  padding-left: 1em;
+  border-left: 1px solid #aaa;
+}
+
+div.yellow-bar {
+  background-color: rgb(255, 221, 51);
+}
+
+div.header-lastrow {
+  display: block;
+  padding-top: 0.5em;
+  padding-bottom: 0.5em;
+}
+
+div.header-lastrow > span {
+  margin-left: 1.5em;
+}
+
+@media (max-width: 935px) {
+  nav.wrapperleft-link > ul {
+    display: none;
+  }
+
+  nav.wrapperright-link > ul {
+    display: none;
+  }
+
+  div.headersecond-row > div > p,
+  div.headersecond-row > nav > ul > li:nth-child(2),
+  div.headersecond-row > nav > ul > li:nth-child(3),
+  div.headersecond-row > nav > ul > li:nth-child(4),
+  div.headersecond-row > nav > ul > li:nth-child(5),
+  div.headersecond-row > nav > ul > li:nth-child(6),
+  div.headersecond-row > nav > ul > li:nth-child(7),
+  div.headersecond-row > nav > ul > li:nth-child(8) {
+    display: none;
+  }
+}
+
+@media (max-width: 932px) {
+  div.headersecond-row > .three-bars {
+    display: inline-block;
+    margin-top: 1em;
+  }
+
+  div.headersecond-row > nav > ul {
+    margin-bottom: 0;
+    margin-top: 0.5em;
+  }
+
+  div.headersecond-row > nav > ul > li > a > img {
+    width: 45%;
+    height: 65%;
+  }
+}
+
+@media (min-width: 933px) {
+  div.headersecond-row nav > ul > li:nth-child(1),
+  div.headersecond-row > .three-bars {
+    display: none;
+  }
+}
+
+div.cover-image {
+  background-image: url(https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/05/1-18-796x417.jpg);
+  height: 23em;
+  width: 100%;
+  margin-left: 0.5em;
+  margin-top: 0.1em;
+  float: left;
+  background-size: cover;
+}
+
+@media (max-width: 797px) {
+  div.cover-image {
+    background-image: url(https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/05/1-18-796x417.jpg);
+    height: 23em;
+    width: 96%;
+    margin-left: 0.5em;
+    margin-right: 0.5em;
+    margin-top: 0.1em;
+    float: left;
+    background-size: cover;
+  }
+}
+
+.wrappersubgrid > div > div:nth-child(1) {
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/05/Sprint_image.jpg");
+  background-size: cover;
+  height: 11.5em;
+  width: 98%;
+  margin-left: 0.5em;
+  margin-bottom: 0.3em;
+  margin-top: 0.1em;
+}
+
+@media (max-width: 600px) {
+  .wrappersubgrid > div > div:nth-child(1) {
+    background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/05/Sprint_image.jpg");
+    background-size: cover;
+    height: 11.5em;
+    width: 96%;
+    margin-left: 0.5em;
+    margin-bottom: 0.3em;
+    margin-top: 0.1em;
+  }
+}
+
```

Comment:
Let's remove the repeated code and make the CSS file clean.

Reply:
ok thanks, I resolved this